<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPanelWidget</class>
 <widget class="QWidget" name="ConfigPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configure panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>539</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_size">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Size</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_length">
               <property name="toolTip">
                <string>&lt;p&gt;Negative pixel value sets the panel length to that many pixels less than available screen space.&lt;/p&gt;&lt;p/&gt;&lt;p&gt;&lt;i&gt;E.g. &quot;Length&quot; set to -100px, screen size is 1000px, then real panel length will be 900 px.&lt;/i&gt;&lt;/p&gt;</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Size:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_length">
               <property name="text">
                <string>Length:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="comboBox_lenghtType">
               <item>
                <property name="text">
                 <string>%</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>px</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QSpinBox" name="spinBox_panelSize">
               <property name="suffix">
                <string> px</string>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_iconSize">
               <property name="suffix">
                <string> px</string>
               </property>
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>128</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_iconSize">
               <property name="text">
                <string>Icon size:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_lineCount">
               <property name="text">
                <string>Rows:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_lineCount">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Alignment &amp;&amp; position</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_alignment">
            <property name="text">
             <string>Alignment:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_alignment">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Left</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Center</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Right</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_position">
            <property name="text">
             <string>Position:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_position"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_hidable">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>A&amp;uto-hide</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_animation">
               <property name="toolTip">
                <string>Zero means no animation</string>
               </property>
               <property name="text">
                <string>Animation duration:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>102</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="spinBox_animation">
               <property name="toolTip">
                <string>Zero means no animation</string>
               </property>
               <property name="suffix">
                <string> ms</string>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="singleStep">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_delay">
               <property name="toolTip">
                <string>Zero means no delay</string>
               </property>
               <property name="text">
                <string>Show with delay:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="spinBox_delay">
               <property name="toolTip">
                <string>Zero means no delay</string>
               </property>
               <property name="suffix">
                <string> ms</string>
               </property>
               <property name="maximum">
                <number>2000</number>
               </property>
               <property name="singleStep">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QCheckBox" name="checkBox_visibleMargin">
               <property name="text">
                <string>Visible thin margin for hidden panel</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="3">
              <widget class="QCheckBox" name="checkBox_overlap">
               <property name="text">
                <string>Hide only on overlapping a window</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_reserveSpace">
            <property name="toolTip">
             <string>Don't allow maximized windows go under the panel window</string>
            </property>
            <property name="text">
             <string>Reserve space on display</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Custom styling</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="5">
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBox_customFontColor">
               <property name="text">
                <string>Font color:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_customFontColor">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="color-picker">
                 <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>5</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_customBgColor">
               <property name="text">
                <string>Background color:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_customBgColor">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="color-picker">
                 <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" colspan="5">
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Background opacity:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_opacity">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="pageStep">
                <number>5</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="5">
           <widget class="QLabel" name="compositingL">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&lt;small&gt;Compositing is required for panel transparency.&lt;/small&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBox_customBgImage">
            <property name="text">
             <string>Background image:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="4">
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lineEdit_customBgImage">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_customBgImage">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="insert-image">
                 <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_icon">
         <property name="toolTip">
          <string>A partial workaround for widget styles that
cannot give a separate theme to the panel.

You might also want to disable:

LXQt Appearance Configuration →
Icons Theme →
Colorize icons based on widget style (palette)</string>
         </property>
         <property name="title">
          <string>Override icon &amp;theme</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Icon theme for panels:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_icon"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_customBgColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_customBgColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_customBgImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_customBgImage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_customBgImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_customBgImage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_customFontColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_customFontColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_customBgColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>slider_opacity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_customBgColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_customBgColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>compositingL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
